nestedbox_core.entity.inc

```diff
@@ -77,6 +77,7 @@ function nestedbox_core_entity_info() {
       'label' => 'label',
     ),
     'access callback' => 'nestedbox_type_access',
+    'static cache' => TRUE,
```

Comment:
*   - static cache: (used by DrupalDefaultEntityController) FALSE to disable
     *     static caching of entities during a page request. Defaults to TRUE.